packages/core/package.json

```diff
@@ -14,12 +14,12 @@
   "bugs": "https://github.com/paritytech/polkadot-onboard.git/issues",
   "homepage": "https://github.com/paritytech/polkadot-onboard.git#readme",
   "license": "Apache-2.0",
-  "main": "dist/index.js",
-  "types": "dist/types/index.d.ts",
-  "type": "module",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "scripts": {
-    "build": "rm -rf ./dist/ && yarn format && tsc --build",
-    "build:watch": "tsc --watch",
+    "build": "rm -rf dist/ && yarn format && yarn build:esm && yarn build:cjs",
+    "build:esm": "tsc --outDir dist/esm",
+    "build:cjs": "tsc --module CommonJS --outDir dist/cjs",
```

Comment:
looking at it again I don't think we want to export .cjs here. This lib is only intended to work in browser runtime, so should be imported as esm.